product/views.py

```diff
@@ -1,3 +1,70 @@
-from django.shortcuts import render
+import json
+import re
+from django.http             import JsonResponse
+from django.views            import View
 
-# Create your views here.
+
+from product.models          import Product, Sale
+from user.models             import User
+from decorators.utils        import login_required
+
+import operator
+
+class ProductListView(View):  
+    # @login_required
+    def get(self,request, catCd, sorting):
```

Comment:
catCd <- 값은 어떤 값인가요?
```suggestion
    def get(self,request, catCd, sorting):
```

Reply:
네 병민님! category id 값입니다! URI parameter로 받아서 해당 카테고리에 해당하는 상품들을 필터링 해주기 위해서 사용했습니다. 조금더 직관적이게 catId식으로 이름을 바꿔 보겠습니다